.mandatory-checkstyle.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!--
+
+  ====
+  Note: This configuration file has been adapted from:
+  https://github.com/checkstyle/checkstyle/blob/master/sun_checks.xml
```

Comment:
This resource was accidentally deleted from their website. However it is restored now. This is a correct link.